supabase/migrations/20260816191358_rls_and_security.sql

```
@@ -0,0 +1,161 @@
+-- plan_workouts
+alter table public.plan_workouts enable row level security;
+create policy "Users can manage their own plan workouts"
+  on public.plan_workouts
+  for all
+  to authenticated
+  using ((select auth.uid())::text = user_id)
+  with check ((select auth.uid())::text = user_id);
+
+-- activities
+alter table public.activities enable row level security;
+create policy "Users can manage their own activities"
```

SQLFluff (4.2.2)

[error] Do not use special characters in identifiers.

(RF05)